File: msgraph-mail/go/utilities/users/inferenceClassification/overrides/overrides_response.go
package overrides
import (
    "github.com/microsoft/kiota-samples/msgraph-mail/go/utilities/users/inferenceClassification"
    "github.com/microsoft/kiota/abstractions/go/serialization"
)
type OverridesResponse struct {
    additionalData Map<String, Object>;
    nextLink *string;
    value []InferenceClassificationOverride;
}

File: msgraph-mail/swiftconsoleapp/Sources/swiftconsoleapp/Users/Item/InferenceClassification/Overrides/OverridesResponse.swift
extension Swiftconsoleapp.Users.Item.InferenceClassification.Overrides {
    public class OverridesResponse : IAdditionalDataHolder {
        public var additionalData: IDictionary<string, object>
        public var nextLink: String
        public var value: [InferenceClassificationOverride]
    }
}
